irr/src/CIrrDeviceSDL.h

```diff
@@ -96,6 +96,9 @@ class CIrrDeviceSDL : public CIrrDeviceStub
 	//! Checks if the Irrlicht device supports touch events.
 	bool supportsTouchEvents() const override;
 
+	//! Checks whether windowing uses the Wayland protocol.
+	bool isUsingWayland() const override;
```

Comment:
A thought: `SomeEnum getDisplayServerType() const override;` would be more generic, thus more future-proof to give specific setups a special treatment.

Reply:
Unlikely that we'll need it so making that "generic" now would be wasted work IMO.